rust/impls/src/postgres_store.rs

```diff
@@ -227,28 +239,30 @@ impl KvStore for PostgresBackendImpl {
 		&self, user_token: String, request: PutObjectRequest,
 	) -> Result<PutObjectResponse, VssError> {
 		let store_id = &request.store_id;
```

Comment:
No need to take a reference here. Without it you move the value, and then calling `to_string` later is equivalent to `clone`. Then you don't need it for the last use because it is moved. Likewise in `delete`. Removing the references would allow addressing the other two comments.